src/js/providers/html5.js

```diff
@@ -368,6 +369,9 @@ define([
                 _setVideotagSource();
                 _videotag.load();
             } else {
+                if (_forceStateToLoad) {
```

Comment:
The state becomes IDLE when we try to replay the same video, which will call the "buffer" event.
I also tried to do it in the model, but it will have to be with when the JWPLAYER_MEDIA_BUFFER_FULL event happens, which will again call the "buffer" event when replayed.